newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -283,3 +283,18 @@ export const onRenderNewProjectSaveAsLocationChooser = ({
     />
   );
 };
+
+export const isTryingToSaveInForbiddenPath = (
+  saveAsLocation: SaveAsLocation
+): boolean => {
+  const fileIdentifier = saveAsLocation.fileIdentifier;
+  if (!fileIdentifier) return false;
+  // If the user is saving locally and chose the same location as where the
+  // executable is running, prevent this, as it will be deleted when the app is updated.
+  const exePath = remote.app.getPath('exe');
```

Comment:
There is a slight risk for remote/remote.app to be null if this is launched on the webapp?
Might be worth adding a check

Reply:
true, never hurts